src/System.Diagnostics.PerformanceCounter/tests/InstanceDataTests.cs

```diff
@@ -0,0 +1,139 @@
+        public static void InstanceDataCollection_GetValues()
+        {
+            InstanceDataCollection idc = GetInstanceDataCollection();
+
+            InstanceData[] values = new InstanceData[idc.Count];
+            idc.Values.CopyTo(values, 0);
+
+            Assert.True(values.Length > 0);
+        }
+
+        [ConditionalFact(typeof(AdminHelpers), nameof(AdminHelpers.IsProcessElevated))]
+        public static void InstanceDataCollectionCollection_GetItem_Invalid()
+        {
+            InstanceDataCollectionCollection idcc = GetInstanceDataCollectionCollection();
+
+            Assert.Throws<ArgumentNullException>(() => idcc[null]);
+        }
+
+        [ConditionalFact(typeof(AdminHelpers), nameof(AdminHelpers.IsProcessElevated))]
+        public static void InstanceDataCollectionCollection_GetKeys()
+        {
+            InstanceDataCollectionCollection idcc = GetInstanceDataCollectionCollection();
+
+            Assert.True(idcc.Keys.Count > 0);
+        }
+
+        [ConditionalFact(typeof(AdminHelpers), nameof(AdminHelpers.IsProcessElevated))]
+        public static void InstanceDataCollectionCollection_GetValues()
+        {
+            InstanceDataCollectionCollection idcc = GetInstanceDataCollectionCollection();
+
+            Assert.True(idcc.Values.Count > 0);
+        }
+
+        [ConditionalFact(typeof(AdminHelpers), nameof(AdminHelpers.IsProcessElevated))]
+        public static void InstanceDataCollectionCollection_Contains_Valid()
+        {
+            InstanceDataCollectionCollection idcc = GetInstanceDataCollectionCollection();
+
+            Assert.True(idcc.Contains("% User Time"));
+        }
+
+        [ConditionalFact(typeof(AdminHelpers), nameof(AdminHelpers.IsProcessElevated))]
+        public static void InstanceDataCollectionCollection_Contains_inValid()
+        {
+            InstanceDataCollectionCollection idcc = GetInstanceDataCollectionCollection();
+
+            Assert.Throws<ArgumentNullException>(() => idcc.Contains(null));
+        }
+
+        [ConditionalFact(typeof(AdminHelpers), nameof(AdminHelpers.IsProcessElevated))]
+        public static void InstanceDataCollectionCollection_Contains_CopyTo()
+        {
+            InstanceDataCollectionCollection idcc = GetInstanceDataCollectionCollection();
+
+            InstanceDataCollection[] idc = new InstanceDataCollection[idcc.Values.Count];
+            idcc.CopyTo(idc, 0);
+            Assert.True(idc.Length > 0);
+        }
+
+        public static InstanceDataCollectionCollection GetInstanceDataCollectionCollection()
+        {
+            PerformanceCounterCategory pcc = new PerformanceCounterCategory("Processor");
+            return pcc.ReadCategory();
+        }
+
+        public static InstanceDataCollection GetInstanceDataCollection()
+        {
+            InstanceDataCollectionCollection idcc = GetInstanceDataCollectionCollection();
+            return idcc["% User Time"];
```

Comment:
You might want to try changing your Windows locale to non English (then log in/out) and verify your tests still pass, that will verify that strings like this are not localized.

Our tests must pass on the machines of contributors who aren't using English.